bin/diagnostic.js

```diff
@@ -1 +1,20 @@
 'use strict';
+
+const ask = require('../lib/ask.js');
+
+let answer = 'not empty';
+
+while (answer !== '') {
+  answer = ask("Guess my secret? ");
+  if (answer === 'SeCrEt') {
+   console.log("You guessed it!");
+   break;
+  }
+}
```

Comment:
I'm not sure if you ran out of time on this one, but please look this over again. Here's one possible solution. If this is unclear to you, please let us know.
```
const ask = require('../lib/ask.js');

let answer = '';

while (answer !== '5') {
  if (answer < 1 || answer > 10) {
   console.log('Please try a number between 1 and 10');
 } else if (answer !== '') {
   console.log('That\'s not it!');
 }
  answer = ask("Guess my secret? ");
}
console.log("You guessed it!");
```